bookkeeper-server/src/main/java/org/apache/bookkeeper/net/NetUtils.java

```diff
@@ -85,4 +86,44 @@ public static String resolveNetworkLocation(DNSToSwitchMapping dnsResolver, Inet
         return rNames.get(0);
     }
 
+    public static String getLocalHost() throws UnknownHostException {
+        String ip = InetAddress.getLocalHost().getHostAddress();
+        if (ip.equals("127.0.0.1")) {
```

Comment:
We should use a better way, this approach does not deal with the whole 127.0.0.0/24 class and with ipv6

I think there is some InteAddress.isLoopback method

Reply:
You are right